src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -15,7 +15,7 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
+    public static final String USERGUIDE_URL = "https://github.com/AY2122S1-CS2103T-T09-1/tp/blob/master/docs/UserGuide.md";
```

Comment:
We have a site for our repo too! Can reference our user guide directly with https://ay2122s1-cs2103t-t09-1.github.io/tp/UserGuide.html instead

Reply:
okay thanks! updated it